contracts/cw20-merkle-airdrop/helpers/README.md

```diff
@@ -26,12 +26,12 @@ Binary will be placed to path.
 
 **Generate Root:**
 ```shell
-merkle-airdrop-cli generateRoot --file ../testdata/airdrop_stage_2.json
+merkle-airdrop-cli generateRoot --file ../testdata/airdrop_stage_2_list.json
 ```
 
 **Generate proof:**
 ```shell
-merkle-airdrop-cli generateProofs --file ../testdata/airdrop_stage_2.json \
+merkle-airdrop-cli generateProofs --file ../testdata/airdrop_stage_2_list.json \
```

Comment:
Ah, did you use the output of these as input to rust tests?

It does look like you have a place to generate test vectors.

Reply:
Yep exactly